File: Python/2006.py
class Solution:
    def countKDifference(self, nums: List[int], k: int) -> int:
        count = 0
        for i in range(len(nums)):
            for j in range(len(nums)):
                if i<j and abs(nums[i] - nums[j]) == k:
                    count += 1
        return count

File: Solutions/Java/2006.java
class Solution {
    public int countKDifference(int[] nums, int k) {
        int count = 0;
        for (int i=0; i<nums.length; i++) {
            for (int j=i+1; j<nums.length; j++) {
                if (Math.abs(nums[j]-nums[i]) == k) {
                    count += 1;
                }
            }
        }
        return count;
    }
}
